group_project/assignment_3.py

```diff
@@ -1,5 +1,8 @@
 def replace_bad_words(a_string, word_to_replace, new_word):
-    pass
+    if word_to_replace not in a_string:
+        return a_string
+    elif word_to_replace in a_string:
+        return a_string.replace(word_to_replace, new_word)
 
```

Comment:
if you stard with this as, replace "elif " by "if" then you can add a line in 6 return a_string, so you take account for the case where word_to_replace is not in a string.